mppm/virtual-players/virtual-players-layout.md

```diff
@@ -0,0 +1,34 @@
+---
+id: virtual-players-layout
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
+
+note:::
+When you exit Play Mode in the Editor, Unity disables the Hierarchy, Inspector, and Scene views in the virtual player window.
```

Comment:
Since people typically launch the virtual players starting in edit mode and usually enter playmode , maybe we should reword this to say that Hierarcy , Inspector and Scene View are only available when you enter playmode or are in playmode.